### PR TITLE
Doc adjustments for v1

### PR DESCRIPTION
**Pull request by jwpereira**
With us heading into the marketplace real soon, I want to make sure the README will reflect the v1 status and latest discussions on naming 🤔

### CODE REVIEW

**Review by acampbel — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ The [Run MATLAB Tests](#run-matlab-tests) GitHub&reg; action enables you to run
 
 - To use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the topmost MATLAB version on the runner's system path.
 
-- To use a GitHub-hosted runner, you must include the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to install MATLAB on the runner. Currently, this action is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to install MATLAB on the runner. Currently, this action is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
```
Comment (mw-hrastega) [inline README.md:7]: install  -> set up

File: README.md [hunk 2]
```diff
@@ -23,11 +23,11 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v2
       - name: Run tests
-        uses: matlab-actions/run-tests@v0
+        uses: matlab-actions/run-tests@v1
 ```
 
 ### Generate Test Artifacts on GitHub-Hosted Runner
-Before you run tests and generate artifacts on a GitHub-hosted runner, first use the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action installs your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
+Before you run tests and generate artifacts on a GitHub-hosted runner, first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action installs your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
```
Comment (mw-hrastega) [inline README.md:30]: installs -> sets up

**Review by mcafaro — APPROVED** [review]
+1 replace all instances of "install" with "set up"

**Review by mw-hrastega — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jwpereira) [reply README.md]: For real this time (I hope): done 😂

File: README.md (hunk 2 above)
Comment (jwpereira) [reply README.md]: For real this time (I hope): done 😂

**Pull request merged**